client/Lykke.Service.Assets.Client/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Lykke.Service.Assets.Client.Cache;
+using Lykke.Service.Assets.Client.Models;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Lykke.Service.Assets.Client
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static void UseAssetsClient(this IServiceCollection services, AssetServiceSettings settings)
```

Comment:
Since this update contains breaking changes anyway, is good time to rename this method to fit common pattern, used in other similar extensions in the Lykke. Pattern is RegisterXXX, so rename it to the RegisterAssetsClient